### ISSUE TITLE
New Rule: Use `select` In Selector names

### ISSUE THREAD

**Issue by prince-chrismc** [timdeschryver/eslint-plugin-ngrx#168]
## Information about the new rule

https://redux.js.org/style-guide/style-guide#name-selector-functions-as-selectthing
Which is inline with NgRx ex https://github.com/ngrx/platform/pull/2151

## A snippet of incorrect code

```ts
export const users = createFeatureSelector<AppState, UsersState>('users');
```

## A snippet of correct code

```ts
export const selectUsers = createFeatureSelector<AppState, UsersState>('users');
```

## I would be willing to submit a PR for the docs :heart:

- [x] Yes (Assistance is provided if you need help submitting a pull request)
- [ ] No

**Comment by timdeschryver** [comment]
🤩 I like it!
If you want, feel free to create a PR!

**Comment by timdeschryver** [comment]
Hey @prince-chrismc, do you want to create a Pull Request to add this rule?

**Comment by prince-chrismc** [comment]
Yes! I started the work but I might need help with the eslint selector 🤔
